feature/gribi/ate_tests/gribi_scaling/gribi_scaling_test.go

```diff
@@ -298,14 +298,15 @@ func createVrf(t *testing.T, dut *ondatra.DUTDevice, d *oc.Root, vrfs []string)
 			i := d.GetOrCreateNetworkInstance(vrf)
 			i.Type = oc.NetworkInstanceTypes_NETWORK_INSTANCE_TYPE_L3VRF
 			i.Enabled = ygot.Bool(true)
-			i.EnabledAddressFamilies = []oc.E_Types_ADDRESS_FAMILY{
-				oc.Types_ADDRESS_FAMILY_IPV4,
-				oc.Types_ADDRESS_FAMILY_IPV6,
-			}
 			i.GetOrCreateProtocol(oc.PolicyTypes_INSTALL_PROTOCOL_TYPE_STATIC, *deviations.StaticProtocolName)
 			gnmi.Replace(t, dut, gnmi.OC().NetworkInstance(vrf).Config(), i)
 			nip := gnmi.OC().NetworkInstance(vrf)
 			fptest.LogQuery(t, "nonDefaultNI", nip.Config(), gnmi.GetConfig(t, dut, nip.Config()))
+		} else if vrf == *deviations.DefaultNetworkInstance {
```

Comment:
@avaneesh90  could you please guard this code block with ExplicitInterfaceInDefaultVRF deviation.

Reply:
@avaneesh90 can you please address the comment?

Reply:
> @avaneesh90 could you please guard this code block with ExplicitInterfaceInDefaultVRF deviation as well.

@mojiiba - Assigning interface to default network instance is part of below code after creating VRF https://github.com/avaneesh90/featureprofiles/blob/TE-14.1_gRIBI_Scaling/feature/gribi/ate_tests/gribi_scaling/gribi_scaling_test.go#L437
Please let us know if you still want to guard this block

Reply:
may be we still require the deviation @mojiiba mentioned.

Reply:
please add the guard, we we still need that. thanks a lot!

Reply:
I do see ExplicitInterfaceInDefaultVRF deviation is used when configuring the default VRF, so the suggestion from @mojiiba should be taken.
However, I don't see Cisco device use this deviation. @mojiiba, can you confirm if this deviation will help Cisco device?